frontend/public/locales/en/dashboard.json

```diff
@@ -25,6 +25,11 @@
     "comments": "Comments & Notes",
     "personal-explanation": "This secret is personal. It is not shared with any of your teammates.",
     "generate-random-hex": "Generate Random Hex",
-    "digits": "digits"
+    "digits": "digits",
+    "delete-key-dialog": {
```

Comment:
this structure is valid?

Reply:
yeah, I think that's good. We will need to modify this to make it more consistent